### PR TITLE
Add withdrawal options and cleanup

### PR DESCRIPTION
**Pull request by lukaszraczylo**
**This pull request:**
- adds withdrawal to specified coinbase wallet.
- adds withdrawal to specified crypto address.
- provides cleaner code based on golint and gosimple outputs.

### CODE REVIEW

**Comment by lukaszraczylo** [comment]
@preichenberger - just bringing it to your attention.

File: withdrawals.go [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+}
+
+func (c *Client) CreateWithdrawalCrypto(newWithdrawCrypto *WithdrawalCrypto) (WithdrawalCrypto, error) {
+	var savedWithdrawal WithdrawalCrypto
+	url := fmt.Sprintf("/withdrawals/crypto")
+	_, err := c.Request("POST", url, newWithdrawCrypto, &savedWithdrawal)
+	return savedWithdrawal, err
+}
+
+func (c *Client) CreateWithdrawalCoinbase(newWithdrawCoinbase *WithdrawalCoinbase) (WithdrawalCoinbase, error) {
```
[the remark this answers is not in the document]
Comment (lukaszraczylo) [reply withdrawals.go]: Can't see this one

File: withdrawals.go [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+	CryptoAddress string `json:"crypto_address"`
+}
+
+type WithdrawalCoinbase struct {
+	Currency          string `json:"currency"`
+	Amount            string `json:"amount"`
+	CoinbaseAccountID string `json:"coinbase_account_id"`
+}
+
+func (c *Client) CreateWithdrawalCrypto(newWithdrawCrypto *WithdrawalCrypto) (WithdrawalCrypto, error) {
```
[the remark this answers is not in the document]
Comment (lukaszraczylo) [reply withdrawals.go]: Can't see this one

[the remark this answers is not in the document]
Comment (preichenberger) [reply withdrawals.go]: The arguments you have: newWithdrawCrypto *WithdrawalCrypto, I think you meant: newWithdrawlCrypto *WithdrawalCrypto)

File: withdrawals.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (preichenberger) [reply withdrawals.go]: The arguments you have: newWithdrawCrypto *WithdrawalCrypto, I think you meant: newWithdrawlCrypto *WithdrawalCrypto)

**Pull request merged**